crates/rpc-server/src/registry.rs

```diff
@@ -726,6 +728,18 @@ async fn compute_l2_sudt_script_hash(
     Ok(to_jsonh256(l2_sudt_script.hash().into()))
 }
 
+async fn get_node_info(generator: Data<Generator>) -> Result<NodeInfo> {
+    let backend = generator
+        .get_backend_info()
+        .into_iter()
+        .map(|backend_info| backend_info.into())
```

Comment:
We calculate blake2b for three binaries every time, which isn't necessary.

Reply:
Let me put this in the struct of `Generator` as a field of backend_info: Vec\<BeckendInfo\>.